File: Programming_Languages/asgn2/python_code/task3_python/Washer.py
# this is the python Washer code

import SmartFixture
import Attendant
import Room

class Washer(SmartFixture):
    def __init__(self,Name,Weight,Position,Battery_capacity):
        super(Washer,self).__init__(Name, Weight, Position, Battery_capacity)

    def Wash(self):
        print('%s finished a wash procedure.' % self._name)

File: Programming_Languages/asgn2/python_code/task3_java/Washer.java
package assg2task3;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author kenchan
 */
public class Washer extends SmartFixture {
    public Washer(String Name, double Weight, int Position, double Battery_capacity){
        super(Name, Weight, Position, Battery_capacity);
    }
    public void Wash(){
        System.out.println(this.name + " finished a wash procedure.");
    }
}
